<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUiAbstractDialog</class>
 <widget class="QDialog" name="CUiAbstractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUiAbstractDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_titleBar" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true"/>
         </item>
         <item>
          <widget class="QLabel" name="m_title">
           <property name="text">
            <string>About JetMind</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnClose">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>12</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/global-close-dialog.png</normaloff>:/res/icons/global-close-dialog.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_main" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
